src/askscript/__tests__/99-e2e-tests/e2e-12-bubble_sort.ask

```diff
@@ -0,0 +1,13 @@
+// Bubble sort
+ask {
+  let arr = [342.34, 35767, 2, 117, 999, 3435]
+  const len = arr:length
+  for (let i = 0; i < len; i = i + 1) {
+	  for (let j = 0; (j + 1) < len; j = j + 1){
```

Comment:
@czerwinskilukasz1 is this right or should it say `j < len`?

Reply:
len its fine but problem is in the const `const len = arr:length` :) should be 'dot' `arr.length`

Reply:
This is AskScript not JavaScript :)
`arr` can be called with `length` so `arr:length`
it's the same as `length(arr)`

Reply:
I would 

> @czerwinskilukasz1 is this right or should it say `j < len`?

I think it should be rather `j < len-i-1`, following https://www.geeksforgeeks.org/bubble-sort/
Of course `j < len` should work too.

Reply:
@czerwinskilukasz1 done